vue/example/index.html

```diff
@@ -20,12 +20,37 @@
 </head>
 
 <body>
-<div class="container">
-  <div class="starter-template">
-    <app></app>
-  </div>
-</div>
+    <div id="app">
```

Comment:
your using four-space indentation here and then two-space indentation below. Can you keep everything at two-space indentation?